src/components/member-components/MembersGrid.tsx

```diff
@@ -0,0 +1,7 @@
+export const MembersGrid = ({ children }) => {
+  return (
+    <div className="grid grid-cols-1  sm:grid-cols-gap  lg:grid-cols-3 xl:grid-cols-4 gap-5 w-fit mx-auto justify-center">
```

Comment:
Talán lehetne md:grid-cols-2, mert pl itt elég széles de mégis 1 oszlopba rendezi (de nézd meg hogy hogyan lesz jó)
![image](https://github.com/user-attachments/assets/09593ec3-2852-43d7-84f3-898cb4a1fc16)

Reply:
Valószínűleg a "sm:grid-cols-gap" az az akart lenni mert még sm-ben is jó a 2 oszlop